app/routes.py

```diff
@@ -16,12 +16,16 @@ def index():
 
 @app.route('/classify', methods=['POST'])
 def classify():
-    # request.args: {'classifier': 'bert_for_countries'}
+    # request.args: {
+    #       'bert': 'bert_uncased_L-12_H-768_A-12/1',
+    #       'classifier': 'bert_for_countries',
+    #       'vocab': 'label.vocab'}
     # request.form: {'seq1', 'seq2', 'seq3'}
     error = None
-    c = classifier.Classifier()
-    c.classify()
-    return "Classification goes here"
+    c = classifier.Classifier(
```

Comment:
I would prefer to move directly to json

Reply:
Requests are get_json()ed now.

Reply:
$ curl --header "Content-Type: application/json" --request POST --data '{"seq":"books are for reading"}' http://localhost:5000/classify
"[('General', 0.99978995), ('Right to education', 0.95538265), ('Rights of the Child', 0.82425225)]"

Reply:
Can I haz playground.http? :)